public/app/features/alerting/unified/components/contact-points/ContactPoints.test.tsx

```diff
@@ -428,5 +449,28 @@ describe('contact points', () => {
 
       return expect(attemptDeleteContactPoint('provisioned-contact-point')).rejects.toBeTruthy();
     });
+
+    it('does not show manage permissions', async () => {
+      renderGrafanaContactPoints();
+
+      await clickMoreActionsButton('lotsa-emails');
```

Comment:
I initially forgot to `await` this click and it was causing all kinds of annoyances in the tests 🙃 
Created https://github.com/grafana/grafana/issues/93405 to enable this